docker-compose.yml

```diff
@@ -24,5 +27,32 @@ services:
       - .parking-service.env
     volumes:
     - images:/cowbot-images
-    #ports:
-    #  - '3000:3000'
+
+  skill-search:
+    build:
+      context: ./skill-search
+    restart: unless-stopped
+    env_file:
+      .skill-search.env
+    volumes:
+      - ./skill-search/data/${SQLITE_DATABASE}:/app/data/${SQLITE_DATABASE}:ro
+
+  job-refresh-skills:
+    build:
```

Comment:
Remove this "build" block and replace it with an `image: fearlesstech/cowbot-skill-search:latest`; this will _use_ the image built with the first service